src/cmd-run

```diff
@@ -186,6 +186,10 @@ fi
 
 set -- -drive if=virtio"${vm_drive_args:-}",file="${VM_DISK}" "$@"
 
+if [ "$(arch)" == "aarch64" ]; then
```

Comment:
Maybe drop a comment here explaining why this is needed? I'm not super familiar with booting on non-x86

Reply:
Sure thing.

Reply:
Added a fixup for that.